README.aspnetcore-build-nightly.md

```diff
@@ -12,12 +12,12 @@ The [.NET Core Docker samples](https://github.com/dotnet/dotnet-docker/blob/mast
 
 # Linux amd64 tags
 
-- [`1.1.11-1.1.12-jessie`, `1.1.11-1.1.12`, `1.1`, `1` (*1.1/jessie/sdk/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/1.1/jessie/sdk/Dockerfile)
+- [`1.1.12-1.1.13-jessie`, `1.1.12-1.1.13`, `1.1`, `1` (*1.1/jessie/sdk/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/1.1/jessie/sdk/Dockerfile)
```

Comment:
@MichaelSimons are these tags still formatted correctly or should they be update to just the sdk version `1.1.13` instead?

Reply:
I think they should remain as is.